addons/mail/static/src/rtc/call_action_list.js

```diff
@@ -35,10 +74,6 @@ export class CallActionList extends Component {
         }
     }
 
-    async onClickRaiseHand(ev) {
-        this.rtc.raiseHand(!this.rtc.state.selfSession.raisingHand);
-    }
-
```

Comment:
In general we avoid removing functions in stable versions, as it's possible some customs rely on them.
Also code cleaning is usually more appropriate in `master`. Changes in stable better be minimal for fixing the bug at hand (no pun intended).